worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -475,6 +475,15 @@ private[celeborn] class Worker(
     partitionLocationInfo.toString
   }
 
+  override def getUnavailablePeers: String = {
+    val sb = new StringBuilder
+    sb.append("==========Unavailable Workers in Worker==========\n")
+    unavailablePeers.asScala.map { case (worker, time) =>
```

Comment:
peer to replace worker for both the top description and the variable name in map case. And use foreach instead of map

Reply:
> peer to replace worker for both the top description and the variable name in map case. And use foreach instead of map

Updated